### DISCUSSION TITLE
群晖如果开启防火墙并打开8080端口，但一会就会出错，关闭防火墙就没有任何问题

### DISCUSSION THREAD

**Discussion by itv3** [messense/aliyundrive-webdav#606] [Q&A]
群晖 7.1开启防火墙并打开8080端口，速度会变慢，最明显的变化是File Station中通过【工具】——【远程连接】挂载的阿里webdav，会变得非常缓慢，点击后半天都刷新不出来目录内容，如果关闭防火墙就会秒刷新。一会Docker 容器 aliyundrive-webdav 意外停止，进入日志中发现有多条类似如下的报错信息：
2022-09-19T02:24:12.026176Z  WARN reqwest_retry::middleware: Retry attempt #2. Sleeping 359.609321ms before the next attempt

是不是还有其它端口需要打开呢

**Comment by Cyiming** [comment]
我直接打不开
